apidef/oas/operation.go

```diff
@@ -314,127 +357,128 @@ func newAllowance(prev **Allowance) *Allowance {
 	return *prev
 }
 
-func (s *OAS) fillTransformRequestMethod(metas []apidef.MethodTransformMeta) {
-	for _, meta := range metas {
-		operationID := s.getOperationID(meta.Path, meta.Method)
-		operation := s.GetTykExtension().getOperation(operationID)
-		if operation.TransformRequestMethod == nil {
-			operation.TransformRequestMethod = &TransformRequestMethod{}
-		}
+func filler[T any](
+	dest *OAS,
+	src []T,
+	mapper *pathnormalizer.Mapper,
+	fn func(T, *Operation),
+) func() error {
+
+	return func() error {
+		typ := reflect.TypeFor[T]()
+		pathField, ok := typ.FieldByName("Path")
 
-		operation.TransformRequestMethod.Fill(meta)
-		if ShouldOmit(operation.TransformRequestMethod) {
-			operation.TransformRequestMethod = nil
+		if !ok || pathField.Type.Kind() != reflect.String {
+			return fmt.Errorf("%s.%s has not filed Path with type string", typ.PkgPath(), typ.Name())
 		}
-	}
-}
 
-func (s *OAS) fillTransformRequestBody(metas []apidef.TemplateMeta) {
-	for _, meta := range metas {
-		operationID := s.getOperationID(meta.Path, meta.Method)
-		operation := s.GetTykExtension().getOperation(operationID)
+		methodField, ok := typ.FieldByName("Path")
```

Comment:
this looks like a typo that's affecting logic, can you please confirm if it's intended or not ?

Reply:
Year you are right it's a typo. Thx.